src/llscan.cc

```diff
@@ -397,11 +417,50 @@ bool FindReferencesCmd::DoExecute(SBDebugger d, char** cmd,
     }
   }
 
+  delete scanner;
+
   result.SetStatus(eReturnStatusSuccessFinishResult);
   return true;
 }
 
 
+char** FindReferencesCmd::ParseScanOptions(char** cmd, ScanType& type) {
```

Comment:
I'm not a huge fan of by-reference arguments. We use it for errors and some of lldb APIs, i.e. where it is inevitable, but let's use regular `*` in other places and pass arguments with `&` before them?